exercises/perfect-numbers/generator/generator.jl

```diff
@@ -0,0 +1,23 @@
+# canonical version = 1.3.0
+using Test
+
+using LazyJSON
+
+j = LazyJSON.value(String(read("generator\\canonical-data.json")))
```

Comment:
Please use `joinpath` to join paths in a cross-platform way. This would fail on non-Windows machines.